src/python/pants/core/util_rules/system_binaries.py

```diff
@@ -282,26 +271,6 @@ def extract_archive_argv(self, archive_path: str, extract_path: str) -> tuple[st
         return (self.path, archive_path, "-d", extract_path)
 
 
-@dataclass(frozen=True)
-class GunzipBinary:
-    python: PythonBinary
-
```

Comment:
The gunzip changes are actually cruft from https://github.com/pantsbuild/pants/pull/18433, which meant to move them, but ended up only adding the new code (and not deleting this)